From 8c7ced988def52654946458a3779e383b27f14f7 Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex.kanavin@gmail.com>
Date: Thu, 14 Feb 2019 17:56:10 +0100
Subject: [PATCH] Work-around thumbnailer cross-compile failure

Install gdk-pixbuf-print-mime-types that's needed for thumbnail
metadata generation.

The upstream issue is https://bugzilla.gnome.org/show_bug.cgi?id=779057

Upstream-Status: Inappropriate [workaround]
Signed-off-by: Alexander Kanavin <alex.kanavin@gmail.com>
---
 thumbnailer/meson.build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/thumbnailer/meson.build b/thumbnailer/meson.build
index e801144..aaafec8 100644
--- a/thumbnailer/meson.build
+++ b/thumbnailer/meson.build
@@ -10,6 +10,7 @@ executable('gdk-pixbuf-thumbnailer',
 gdk_pixbuf_print_mime_types = executable('gdk-pixbuf-print-mime-types',
                                          'gdk-pixbuf-print-mime-types.c',
                                          c_args: common_cflags,
+                                         install: true,
                                          dependencies: gdk_pixbuf_deps + [ gdkpixbuf_dep ])
 
 custom_target('thumbnailer',
